### ISSUE TITLE
Multiple images viewer

### ISSUE THREAD

**Issue by zackhsuan** [Julioacarrettoni/EXPhotoViewer#4]
Awesome lib!

It would be even more awesome if you could make it possible to view multiple images in a scroll view.

e.g.
images = [self getAllImages]; // array of images

[EXPhotoViewer showImagesFor:images];

then the original imageview stays at its old place but the other pretty effects still work (zooming, pull-back etc.) and you can view the images by just sliding left and right.

Very appreciate for all your great work!

**Comment by Julioacarrettoni** [comment]
Thanks for the kind words!

Yes, I'm currently working on that, I'm playing with several "dismiss" animations, cause what should happens when you go back?
Should it show the original image going back to its place or the last image you was seeing? but in that case if you have a collection or tableview the cell showing the imageView won't even exists, so it should require the images AND the indexPaths and also scroll to that item when you scroll through images? :|

**Comment by zackhsuan** [comment]
Thanks for your reply!

It was amazing that you are already working on that! lol

That was a good point I think. Here's what I think:
By default, the dismiss action will just dismiss the photoViewer, we dont
need to worry about which photo it should go to.
We can take a normal photo viewer as an example: user clicks on the image,
it either pushed into the navigation controller or presented from the
existing controller. Whichever user is viewing the image, when clicks back
or dismiss, it always goes back the original view controller as where the
clicked image was.

Does that make sense? and also save time for programmers.

Cheers
Zack

**Comment by Julioacarrettoni** [comment]
Yes it makes sense, I'm just trying to achieve something extra, but you are right, I'll wrap this thing up and push my commits.

**Comment by zackhsuan** [comment]
Coool. Appreciate all your work. :)

**Comment by thomasconner** [comment]
Was this every done?

**Comment by emiliomarin** [comment]
Any news on this?

**Comment by adamhongmy** [comment]
It is been 3 years since then. Any news? :D
